datasets/twitter-dataset-graph/Dockerfile

```diff
@@ -1,16 +1,15 @@
-FROM ubuntu:14.04
+FROM cloudsuite/debian:base-os
 #socialcomputing.asu.edu/uploads/1296759055/Twitter-dataset.zip
 # Install packages
 RUN BUILD_DEPS="wget unzip" \
     && apt-get update && apt-get install -y --no-install-recommends ${BUILD_DEPS} \
     && rm -rf /var/lib/apt/lists/* \
     && mkdir -p /data \
-    && wget --no-check-certificate -O /data/Twitter-dataset.zip socialcomputing.asu.edu/uploads/1296759055/Twitter-dataset.zip \
+    && wget --no-check-certificate -O /data/Twitter-dataset.zip https://cloudsuite.ch/download/graph_analytics/twitter_dataset.zip \
     && unzip -d /data /data/Twitter-dataset.zip \
     && rm /data/Twitter-dataset.zip \
-    && sed -i "s/,/ /g" /data/Twitter-dataset/data/edges.csv \
     && mv /data/Twitter-dataset/data/edges.csv /data/edges.csv \
```

Comment:
I believe this path is incorrect. There is no directory called `data` inside the zip file.

Reply:
updated.